### ISSUE TITLE
Fossil records could be shown on the maps of ancient, not present Earth

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3055]
**Fossil records could be shown on the maps of ancient, not present Earth**

https://dinosaurpictures.org/ancient-earth#120

-----

Github user: @dschigel
User: [See in registry](https://www.gbif.org/api/feedback/user/d6ba5a1d4c9cfaf8a1b4a0f6722e0336:231479724afa4c84ca9affb328b2af3d489a8716be0c0f14a45fb57d4eb21e58870d435ec4e4c9f3f0f45eea76281639ec65eb3df0841f3c7d2e301f539bc7eb)
System: Chrome 86.0.4240 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/map?basis_of_record=FOSSIL_SPECIMEN&occurrence_status=present
Window size: width 1920 - height 1067
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-10-18T07:44:24.408Z',mode:absolute,to:'2020-10-18T07:50:24.408Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-10-18T07:44:24.408Z',mode:absolute,to:'2020-10-18T07:50:24.408Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
Yes I've been wanting that as well! There are 2 things to it as far as I can see

1. we need basemap(s) that someone is willing to maintain and share with us.
2. we need an algorithm to convert the coordinates to a different time and that should be used as part of our processing

I've kept an eye on it for [years](https://github.com/gbif/portal16/issues/7#issue-159403669), but no one has ever asked for it. So perhaps it is just you and I @dschigel that think it would be fun. Some other portals do have it though, but I have no idea how useful it is honestly.

-------

More broadly it would be nice with more basemaps:
* ocean depths, salinity, temperatures, weather
* geologic maps
* historic maps (both 100 years old and 100 mil years old and everything in between)
* administrative areas (we have GADM now) and bio/geographic areas.

And ideally they are all global (including oceans) and free. They are not easy to find. I've tried the best i could. GADM is the only global source that I found back when searching first in 2016, and that doesn't include oceans. If you know good complete sources, then please mention them.

**Comment by dschigel** [comment]
With any new feature development, it would be useful to have some sense of user demands; our paleontology community of users can be bigger, and some contacts can be found in the GBIF using literatures, but it is a bit of chicken and the egg problem: maybe such users would come should you have ancient maps... We definitely need to have an analytical / research purpose for the feature, if this just a decorative feature, than it is not needed. Communication / PR value alone probably does not justify experiments.

**Comment by tucotuco** [comment]
Sensible, @Dmitry Schigel [GBIF] <dschigel@gbif.org>  but it would be
sooooooo cool.

As for other sources, I feel I'm being a traitor by mentioning it, but
there is a newer alternative to GADM, geoBoundaries, to be found at
https://www.geoboundaries.org/. I have not assessed in practice yet.
